docs/Deprecating-Disabling-and-Removing-Formulae.md

```diff
@@ -42,16 +44,17 @@ A formula should be disabled to indicate to users that the formula cannot be use
 
 The most common reasons for disabling a formula are:
 
-- it cannot be built from source (meaning no bottles can be built)
+- it cannot be built from source (meaning no new bottles can be built)
```

Comment:
```suggestion
- it cannot be built from source on all supported OS versions (meaning no new bottles can be built)
```

So that we don't instant disable a formula because it fails on one OS.